src/python/grpcio/grpc/_channel.py

```diff
@@ -24,6 +24,8 @@
 from grpc._cython import cygrpc
 from grpc.framework.foundation import callable_util
 
+LOGGER = logging.getLogger(__name__)
```

Comment:
The lack of a leading underscore on this name indicates that it is "module-public" and is available for use outside this module. Is that a thing that we should want? What breaks or goes wrong when we name this `_LOGGER`?

Reply:
Hey. Sorry for the much-delayed response. This was an oversight on my part. You're correct in that we don't need these loggers to be accessible outside the module, so `_LOGGER` makes more sense. I'll make that change.